### PR TITLE
more utils

### PR DESCRIPTION
**Pull request by levinzonr**

### CODE REVIEW

File: android/src/main/java/dk/nodes/utils/android/text/SpannedExtensions.kt [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package dk.nodes.utils.android.text
+
+import android.os.Build
+import android.text.*
+import android.text.style.ClickableSpan
+import android.view.View
+
+fun String.fromHtml(): Spanned {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
```
Comment (markusrubey) [inline android/src/main/java/dk/nodes/utils/android/text/SpannedExtensions.kt]: Let's use `HtmlCompat.fromHtml(html, HtmlCompat.FROM_HTML_MODE_LEGACY)` here :)

File: android/src/main/java/dk/nodes/utils/android/text/SpannedExtensions.kt [hunk 2]
```diff
@@ -0,0 +1,38 @@
+package dk.nodes.utils.android.text
+
+import android.os.Build
+import android.text.*
+import android.text.style.ClickableSpan
+import android.view.View
+
+fun String.fromHtml(): Spanned {
```
Comment (markusrubey) [inline android/src/main/java/dk/nodes/utils/android/text/SpannedExtensions.kt]: Currently maybe overkill but when this lib get's bigger I would prefer to have this in a separate `StringExtensions.kt` file. This file could be named `SpannedStringExtensions.kt` then, what do you think?

File: android/src/main/java/dk/nodes/utils/android/text/SpannedExtensions.kt (hunk 1 above)
Comment (levinzonr) [reply android/src/main/java/dk/nodes/utils/android/text/SpannedExtensions.kt]: lul, thats new :)
Sure lets use that

File: android/src/main/java/dk/nodes/utils/android/text/SpannedExtensions.kt (hunk 2 above)
Comment (levinzonr) [reply android/src/main/java/dk/nodes/utils/android/text/SpannedExtensions.kt]: Makes sense and an easy update, done 👍

File: android/src/main/java/dk/nodes/utils/android/text/SpannedExtensions.kt (hunk 1 above)
Comment (markusrubey) [reply android/src/main/java/dk/nodes/utils/android/text/SpannedExtensions.kt]: I didn't mind `fun String.fromHtml()` as it is way less wordy. Added it again in https://github.com/nodes-android/utils/pull/3/commits/1ceeed8bbf0a41ac8f431b4e5fabb3746865fb48

**Pull request merged**